lib/jekyll/admin.rb

```diff
@@ -1,7 +1,25 @@
 require "jekyll/admin/version"
-
+require "jekyll/admin/admin_servlet"
+require "jekyll/admin/api_servlet"
+require "jekyll/admin/api_handler"
+require "jekyll/admin/configuration/handler"
+require "jekyll/admin/configuration/servlet"
+require "jekyll/admin/pages/handler"
+require "jekyll/admin/pages/servlet"
+require_relative "commands/serve"
 module Jekyll
   module Admin
-    # Your code goes here...
+    # Path to the compiled frontend folder
+    def self.public_path
+      File.join(File.dirname(Dir.pwd), 'jekyll-admin/lib/jekyll/admin/public/src').to_s
```

Comment:
How are we going to put them there? Are we going to manually compile them before merging to master? Also, @mertkahyaoglu, can you add instructions on how to compile the files locally. I was trying to do that but ended up with a blank page (the start `npm start` works fine)